content/source/docs/cloud/api/variable-sets.html.md

```diff
@@ -0,0 +1,645 @@
+        ]
+      }
+    }
+  }
+}
+```
+
+## Update a Variable Set
+
+`PUT/PATCH varsets/:varset_id`
+
+| Parameter            | Description         |
+| -------------------- | --------------------|
+| `:varset_id`         | The variable set ID |
+
+Trying to save 2 variables of the same name and category among any global variable sets will result in a 422 response.
+
+Status  | Response                                     | Reason(s)
+--------|----------------------------------------------|----------
+[200][] | [JSON API document][]                        | Successfully updated variable set
+[404][] | [JSON API error object][]                    | Organization or variable set not found, or user unauthorized to perform action
+[422][] | [JSON API error object][]                    | Problem with payload or request; details provided in the error object
+
+
+### Sample Payload
+
+```json
+{
+  "data": {
+    "type": "varsets",
+    "attributes": {
+      "name": "MyVarset",
+      "description": "Full of vars and such for mass reuse. Now global!",
+      "is-global": true
+    },
+    "relationships": {
+      "vars": {
+        "data": [
+          {
+            "type": "vars",
+            "attributes": {
+              "key": "c2e4612d993c18e42ef30405ea7d0e9ae",
+              "value": "8676328808c5bf56ac5c8c0def3b7071",
+              "category": "terraform"
+            }
+          }
+        ]
+      }
+    }
+  }
+}
+```
+
+### Sample Request
+
+```shell
+curl \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  --request PATCH \
+  https://app.terraform.io/api/v2/varsets/varset-kjkN545LH2Sfercv
+```
+
+### Sample Response
+
+```json
+{
+  "data": {
+    "id": "varset-kjkN545LH2Sfercv"
+    "type": "varsets",
+    "attributes": {
+      "name": "MyVarset",
+      "description": "Full of vars and such for mass reuse. Now global!",
+      "is-global": true
+    },
+    "relationships": {
+      "vars": {
+        "data": [
+          {
+            "id": "var-Nh0doz0hzj9hrm34qq"
+            "type": "vars",
+            "attributes": {
+              "key": "c2e4612d993c18e42ef30405ea7d0e9ae",
+              "value": "8676328808c5bf56ac5c8c0def3b7071",
+              "category": "terraform"
+            }
+          }
+        ]
+      }
+    }
+  }
+}
+```
+
+## Delete a Variable Set
+`DELETE varsets/:varset_id`
+
+| Parameter            | Description         |
+| -------------------- | --------------------|
+| `:varset_id`         | The variable set ID |
+
+### Sample Request
+
+```shell
+curl \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  --request DELETE \
+  https://app.terraform.io/api/v2/varsets/varset-kjkN545LH2Sfercv
+```
+
+On success, this endpoint responds with no content.
+
+## Show Variable Set
+
+Fetch details about the specified variable set.
+
+`GET varsets/:varset_id` #same response sans pagination and listing
```

Comment:
the same response is confusing to me - what does that mean?

```suggestion
`GET varsets/:varset_id` #same response sans pagination and listing
```

Reply:
Oh good catch. Removed.